final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+from Handler import Handler
+
+import logging
+
+
+def main():
+    arg_parser = argparse.ArgumentParser(description="Pure Python command-line RSS reader.")
+    arg_parser.add_argument("source", type=str, help="RSS URL")
+    arg_parser.add_argument("--version", action="store_true", help="Print version info")
+    arg_parser.add_argument("--json", action="store_true", help=" Print result as JSON in stdout")
+    arg_parser.add_argument("--verbose", action="store_true", help="Outputs verbose status messages")
+    arg_parser.add_argument("--limit", type=int, default=1, help="Limit news topics if this parameter provided")
+
+    arg_parser_args = arg_parser.parse_args()
```

Comment:
It is better to move this to a separate function

Reply:
fixed